### DISCUSSION TITLE
Block.IO problem

### DISCUSSION THREAD

**Discussion by alirdashti** [dogecoin/dogecoin#3097] [Help needed]
Dear Sir,

I have an account in [Block.IO](https://block.io/) from 2016 and about 3 months ago they blocked my account without any reason suddenly...!!!

I sent more than 30 emails to their support email but did not receieve any feedback till now.

Please help me , i have about 4000 doge coin in my account , also i have my BIP32 public and private key and second key too.

Moreover i have my Mnemonic 12 words but when using all of them in recover app such as BIP39 standalon or dogecore wallet and etc , i can't recover my wallet.

for example when i use second private key in dogecore or dogechain , they tell me that this wallet address is emty while when check my wallet address in sochain , i can see my amount

this is my wallet address: 9uDEjxn9Lt1dr6wKfFFKYwPnH6DfVvx1zn

please tell me how to recover my wallet with using these data.

Thanks in advance.

**Comment by mooha12** [comment]
Hello brother, I checked your message and it seems that I can recover it, but there is one thing, which is the phrase “12 seeds” if you have them. I checked it today and it contains 3900From his work, Dougie is happy to write to you.

**Comment by mooha12** [comment]
Do you want to recover it, brother? Email me and I will send it to you.

في السبت، ١٣ أغسطس ٢٠٢٢، ٢:٣٣ م alirdashti ***@***.***> كتب:

**Comment by alirdashti** [comment]
Hello brother

Hope you are fine.

Yes this is true ( 3900 ) and i couldn’t recover it till now , and has 12
word .
Can you open it?

Thank you very much for following.

Best regards

**Reply by mooha12** [reply]
I hope you will contact me via Telegram and I will help you, brother 
@mooha100
